### ISSUE TITLE
Admonitions in BLorient11719

### ISSUE THREAD

**Issue by CarstenHoffmannMarburg** [BetaMasaheft/Documentation#2566]
@DenisNosnitsin1970 @eu-genia @karljonaskarlsson 

I found texts in BLorient11719, that Strelcyn called "A chapter of admonitions, in Amharic". I do not know, if there is possibly a connection to LIT4014Medan, that is attested in part 1 of this composite manuscript (the composition is not entirely clear from the catalogue description). However, I did not find the content on "śǝna fǝṭrat" in EMIP02346, that served as witness to me.

Should I create a new LIT ID for these admonitions?

```
<msItem xml:id="p2_i3">
                                <locus from="58vb" to="65v"/><locus from="72r" to="72v"/><locus from="79r" to="80v"/> 
                                <title ref="LIT0000000000000000000000000000000000">Admonitions</title>
                                <msItem xml:id="p2_i3.1">
                                    <locus from="58vb" to="65v"/><locus from="72r"/>
                                    <title ref="LIT0000000000000000000000000000000000">Admonitions</title>
                                <incipit xml:lang="am">ክፍል፡ ዘምዕዳን፡ ስታጭድም፡ ቀርምያ፡ አታጽራ፡ አለ። ቁሞ፡ ሲለምን፡ የለም፡ ቢል፡ እንዴት፡ ድንቅ፡ ይሆን፡ እግር፡ እፀቡ፡ አለ፡
                                    ያ፡ ቀርቶ፡ ለተራበ፡ ባይስጥ፡ እንዴት፡ ድንቅ፡ ይሆን፡ አህያ፡ አጋድሎ፡ ብታይ፡ አጨጭነው፡ አለ፡ ለአህያው፡ ራራቶ፡ ለባለመከራው፡ ሰው፡ አለመርዳት፡ እንዴት፡
                                    ድንቅ፡ ይሆን፡ </incipit>
                                <textLang xml:lang="am"/>
                                </msItem>
                                <msItem xml:id="p2_i3.2">
                                    <locus from="72v"/>
                                    <title ref="LIT00000000000000">Another chapter of the mǝʾdān on the beauty of the creation (sǝnna fǝṭrat)</title>
                                    <incipit xml:lang="am">ክፍል፡ ስነ፡ ፍጥረትም፡ አንዲህ፡ ነው፡ እሑድ፡ በመጀመርያ፡ መሬትን፡ ውሀን፡ እሳትን፡ ነፋስን፡ ፈጠረ፡ ከለዚህ፡ ፍጥረታትን፡
                                        ሁሉ፡ ፈጠረ።</incipit>
                                    <textLang xml:lang="am"/>
                                </msItem>
                            </msItem>
```

**Comment by CarstenHoffmannMarburg** [comment]
I could not identify these texts in LIT4014Medan as attested in EMIP02346. However, since it is a longer text, I cannot say for sure, whether I have not overlooked something. I think, I will create some new LIT IDs for them.

**Comment by CarstenHoffmannMarburg** [comment]
I found no attestations for LIT1615Homily and so, I could not check.

**Comment by CarstenHoffmannMarburg** [comment]
I discovered that one of the admonitions (sena fetratem) is also listed in a catalogue of manuscripts in Cambridge (Ullendorff 1961, no. 43, f. 39a) as part of . It is therefore very likely that both admonitions are part of LIT4014Medan. I am sorry that I overlooked this in Strelcyns catalogue and created three unnecessary IDs.

<img width="338" alt="Screenshot 2024_Medan1" src="https://github.com/BetaMasaheft/Documentation/assets/40291787/a222640d-4bf7-40a9-90af-f99ecaa3b8b0">

![Screenshot 2024-07-11 123520](https://github.com/BetaMasaheft/Documentation/assets/40291787/15f4c4b2-8c7f-482a-96c6-5aca3a5884f2)
